### PR TITLE
removed two broken links, added note

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user issue #1704 , so i removed two broken links and added note

### CODE REVIEW

**Comment by illfated** [comment]
Forgive me for my possible ignorance on the topic matter, but #1704 is a PR with no obvious "issue" as far as I can tell. Did you mean to link an open issue ticket?

**Comment by illfated** [comment]
Never mind, I found the correct issue ticket:
- **Broken Links on the page** #9304

File: windows/deployment/update/windows-update-resources.md [hunk 1]
```diff
@@ -39,9 +39,12 @@ The following resources provide additional information about using Windows Updat
...
+**Reset Windows Update Client settings script** will completely reset the Windows Update client settings. It has been tested on Windows 7, 8, 10, and Windows Server 2012 R2. It will configure the services and registry keys related to Windows Update for default settings. It will also clean up files related to Windows Update, in addition to BITS related data.
 
-[Reset Windows Update Agent script](https://gallery.technet.microsoft.com/scriptcenter/Reset-Windows-Update-Agent-d824badc) allows you to reset the Windows Update Agent, resolving issues with Windows Update.
+**Reset Windows Update Agent script** allows you to reset the Windows Update Agent, resolving issues with Windows Update.
+
+> [!NOTE]
+> Technet gallery hosting site  has been removed, so our Microssoft documentaion Team is working on new hosting site to provide better services, so at persent we  removed two links of **Reset Windows Update Client settings script** and **Reset Windows Update Agent script**.
```
Comment (illfated) [inline windows/deployment/update/windows-update-resources.md:47]: Forgive me for being pessimistic about this PR, but it seems less than productive to remove links, even if they are broken, when the MS Docs team members are working on a new Microsoft-based hosting solution for the content. I think they are going to prefer to let the page stay unchanged until there is a working page alternative to the removed pages.

**Review by JohanFreelancer9 — COMMENTED** [review]
I share @illfated's view and do not think this PR should be merged. I'll hold off on editing until there is clarity in this regard.

**Comment by jaimeo** [comment]
I am fixing this doc today. Do not merge this.

**Pull request closed without merging**